sw/airborne/modules/orange_avoider/orange_avoider.c

```diff
@@ -30,6 +31,31 @@
 #define VERBOSE_PRINT(...)
 #endif
 
+#ifndef ORANGE_AVOIDER_LUM_MIN
```

Comment:
these parameters should be described in the documentation of the module

Reply:
sure

Reply:
fixed